spec/factories/responses.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :response do
+    text "MyString"
+    created_at "2016-10-31 16:25:15"
```

Comment:
hmm ya, maybe we should just make `created_at nil` for now. Are you thinking it might be less confusing to test with if we just make our own created_at datetime for each test?